### PR TITLE
challenge3

### PR DESCRIPTION
**Pull request by Shindha77**

### CODE REVIEW

File: ch3.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+$books['Prince Othon'] = '1885';
+$books['L\'ile au trésor'] = '1883';
+$books['La flèche noire'] = '1888';
```
Comment (aghore97) [inline ch3.php:5]: $books = [
    'Prince Othon' => '1885',
    'L\'ile au trésor' => '1883',
    'La flèche noire' => '1888'
];

File: ch3.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+asort($books);
+foreach ($books as $key => $book){
```
Comment (aghore97) [inline ch3.php:8]: empty line before foreach

File: ch3.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  echo $book . ' - ' . $key;
+}
+
+?>
```
Comment (aghore97) [inline ch3.php:12]: no need for ?>

Comment (Shindha77) [reply ch3.php:12]: Ok, thank you very much :)

**Pull request merged**